coffeecard/CoffeeCard.WebApi/Services/EmailService.cs

```diff
@@ -17,6 +17,7 @@ public class EmailService : IEmailService
         private const string MailgunApiUrl = "https://api.mailgun.net/v3";
```

Comment:
Not on you, but it would be nice if this constant was moved to App Settings

Reply:
`MailgunSettings` that would be

Reply:
Moved there now. We'll need to remember updating the setting in our environments and in the vault